src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/service/TokenServiceTest.java

```diff
@@ -184,6 +192,20 @@ public void deactivateToken() throws Exception {
         }
     }
 
+    @Test
+    @WithMockUser(authorities = { "EDIT_KEYTABLE_FRIENDLY_NAMES", "VIEW_KEYS" })
+    public void updateTokenFriendlyName() throws Exception {
+        TokenInfo tokenInfo = tokenService.getToken(GOOD_TOKEN_ID);
+        assertEquals(GOOD_TOKEN_NAME, tokenInfo.getFriendlyName());
+        try {
+            tokenService.updateTokenFriendlyName(TOKEN_NOT_FOUND_TOKEN_ID, "new-name");
+        } catch (TokenService.TokenNotFoundException expected) {
+            // noop
+        }
```

Comment:
Perhaps create a separate test for calling updateTokenFriendlyName with unexisting token id. This block doesn't give any additional value here.

Reply:
Created a new test and cleaned this one.